components/search/list-renderer/open-reading-frame.js

```diff
@@ -0,0 +1,42 @@
+// node_modules
+import PropTypes from "prop-types";
+// components/search/list-renderer
+import {
+  SearchListItemContent,
+  SearchListItemMain,
+  SearchListItemMeta,
+  SearchListItemQuality,
+  SearchListItemTitle,
+  SearchListItemType,
+  SearchListItemUniqueId,
+} from "./search-list-item";
+
+export default function OpenReadingFrame({ item: openReadingFrame }) {
+  const geneId = openReadingFrame.gene?.map((gene) => gene.geneid).join(", ");
```

Comment:
`gene` is required, so you can just to `openReadingFrame.gene.map`. Too bad it’s not called `genes`. If you don’t feel like removing the question mark, it won’t hurt — it’s just not necessary.

Reply:
done